tests/foreman/ui/test_multinetwork.py

```diff
@@ -0,0 +1,717 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Multi-Network Support feature"""
+from robottelo.test import UITestCase
+from robottelo.common.decorators import stubbed
+
+
+class Multinetwork(UITestCase):
+    """Implements Multi Network support tests in UI."""
+
+    @stubbed()
+    def test_create_host_1(self):
+        """@Test: Create host with default interface and set 'DHCP' for IPAM
+        and BootMode for provisioning subnet
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set 'DHCP' for IPAM and BootMode for provisioning subnet
+
+        2. All other fields on subnet page should be filled
+
+        @Assert: Host should be provisioned with correct configuration under
+         /etc/sysconfig/network-scripts/ifcfg-<interface>
+
+        @Status: Manual
+
+        """
+
+    @stubbed()
+    def test_create_host_2(self):
+        """@Test: Create host with default interface when IPAM set as
+        'Internal DB' (without specifying start and end range) and BootMode
+        set as 'DHCP' for provisioning subnet
+
+        @Feature: Multi Network Support
+
+        @Setup: Provisioning should be configured
+
+        @Steps:
+
+        1. Set IPAM with 'Internal DB' and BootMode with 'DHCP for provisioning
+        subnet
+
+        2. All other fields on subnet page should be filled except start and
+        end IP range
```

Comment:
Make sure to indent all text in a list. You can also drop the lines between list items.

```
1. Set IPAM with 'Internal DB' and BootMode with 'DHCP for provisioning
   subnet
2. All other fields on subnet page should be filled except start and
   end IP range
```